tests/manual/test-procedure.rst

```diff
@@ -404,6 +404,18 @@ Test Case: Autocomplete
   eg type 'di' and be offered 'display'.
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
+Test Case: Incompatible File Handling
```

Comment:
There are some puppeteer tests for checking an error is shown for an invalid hex file (no Python code found inside it) , we can create new tests similar to that one to check all the steps listed here via CI.
https://github.com/bbcmicrobit/PythonEditor/blob/6929c3ba85c4b43dbb7e4e367c861248e3995e03/tests/spec/puppeteer-spec.js#L35-L55

Reply:
Shouldn't this be done in the PR I'm going to make for identifying makecode hex files? are mpy files hex files?

Reply:
The example provided is dealing with hex files, but can create new tests that check what happens when .mpy files or any other type format file is loaded.